tests/pact/fqns.repl

```diff
@@ -77,3 +77,74 @@
 
 (expect "selects correct test" (modB.get-test) "hello")
 (commit-tx)
+
+;;
+;; Module redeploy name resolution
+;;
+
+(begin-tx)
+(namespace 'free)
+(module test-mod-redeploy-ref g
+  (defcap g () true)
+
+  (defcap test ()
+    (enforce false "boom"))
+
+  (defun f ()
+    (with-capability (test)
+      1))
+  )
+(expect-failure "Execution fails due to the falsy defcap" (f))
+(expect-failure "Execution fails due to the falsy defcap" (test-mod-redeploy-ref.f))
```

Comment:
Failure messages should explain _why_ a test is important, not what it's doing. Obviously these three calls fail because the defcap fails. The questions to be answered with the strings are:
- why are we calling this three different ways? copypasting a failure doc doesn't aid comprehension.
- why is it important that we have a failure case, as opposed to calling the defcap and seeing it succeed? An example comment would be "Verify failure to show XXX"

Reply:
Thank you (again) for highlighting it :)